lib/src/code/code_edit_result.dart

```diff
@@ -24,12 +27,18 @@ class CodeEditResult {
   int get hashCode => Object.hash(
         fullTextAfter.hashCode,
         linesChanged.hashCode,
+        indexesChanged.hashCode,
       );
 
   @override
   bool operator ==(Object other) {
     return other is CodeEditResult &&
         fullTextAfter == other.fullTextAfter &&
-        linesChanged == other.linesChanged;
+        linesChanged == other.linesChanged &&
+        indexesChanged == other.indexesChanged;
   }
 }
+
+extension TextRangeExtension on TextRange {
+  int get length => end - start;
+}
```

Comment:
We already have an extension in `/lib/src/code/text_range.dart`
Please add this getter there.